src/libs/Navigation/AppNavigator/Navigators/RightModalNavigator.js

```diff
@@ -1,100 +1,127 @@
-                options={{
-                    // Disable animation for this screen because it causes an animation glitch when using shortcuts
-                    animationEnabled: false,
-                }}
-            />
-            <Stack.Screen
-                name="Details"
-                component={ModalStackNavigators.DetailsModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Profile"
-                component={ModalStackNavigators.ProfileModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Report_Details"
-                component={ModalStackNavigators.ReportDetailsModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Report_Settings"
-                component={ModalStackNavigators.ReportSettingsModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Report_WelcomeMessage"
-                component={ModalStackNavigators.ReportWelcomeMessageModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Participants"
-                component={ModalStackNavigators.ReportParticipantsModalStackNavigator}
-            />
-            <Stack.Screen
-                name="MoneyRequest"
-                component={ModalStackNavigators.MoneyRequestModalStackNavigator}
-            />
-            <Stack.Screen
-                name="NewTask"
-                component={ModalStackNavigators.NewTaskModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Task_Details"
-                component={ModalStackNavigators.TaskModalStackNavigator}
-            />
-            <Stack.Screen
-                name="EnablePayments"
-                component={ModalStackNavigators.EnablePaymentsStackNavigator}
-            />
-            <Stack.Screen
-                name="SplitDetails"
-                component={ModalStackNavigators.SplitDetailsModalStackNavigator}
-            />
-            <Stack.Screen
-                name="AddPersonalBankAccount"
-                component={ModalStackNavigators.AddPersonalBankAccountModalStackNavigator}
-            />
-            <Stack.Screen
-                name="Wallet_Statement"
-                component={ModalStackNavigators.WalletStatementStackNavigator}
-            />
-            <Stack.Screen
-                name="Flag_Comment"
-                component={ModalStackNavigators.FlagCommentStackNavigator}
-            />
-            <Stack.Screen
-                name="EditRequest"
-                component={ModalStackNavigators.EditRequestStackNavigator}
-            />
-        </Stack.Navigator>
+        <>
+            {!isSmallScreenWidth && <Overlay onPress={props.navigation.goBack} />}
+            <View
+                style={[
+                    styles.RHPNavigatorContainer,
+                    {
+                        width: isSmallScreenWidth ? '100%' : variables.sideBarWidth,
```

Comment:
As mentioned earlier we should remove inline styles, and move them to style.js, please checkout `navigationModalCard`